cpp/src/hdbscan/runner.h

```diff
@@ -24,8 +24,8 @@
 #include <cuml/cluster/hdbscan.hpp>
 #include <cuml/common/logger.hpp>
 
-#include <raft/cluster/detail/agglomerative.cuh>
-#include <raft/cluster/detail/mst.cuh>
+#include <raft/cluster/detail/agglomerative.cuh>  // build_dendrogram_host
```

Comment:
Oh boy- detail APIs. Well need to make sure we add this to the tracker list. The single-linkage implementation from RAFT has been migrated to cuVS so we *should* be able to use that. No need to do that for 24.10, though.